token-validation-spring-test/src/main/java/no/nav/security/token/support/spring/test/MockLoginController.java

```diff
@@ -60,4 +59,41 @@ public Cookie addCookie(
         }
         return cookie;
     }
+
+    @Unprotected
+    @PostMapping("/cookie")
+    public Cookie addCoaaokie(
+        @RequestParam(value = "issuerId") String issuerId,
+        @RequestParam(value = "audience") String audience,
+        @RequestParam(value = "subject", defaultValue = "12345678910") String subject,
+        @RequestParam(value = "cookiename", defaultValue = "localhost-idtoken") String cookieName,
+        @RequestParam(value = "redirect", required = false) String redirect,
+        @RequestParam(value = "expiry", required = false) String expiry,
+        @RequestBody Map<String, Object> claims,
+        HttpServletResponse response
+    ) throws IOException {
+        String token =
+            mockOAuth2Server.issueToken(
```

Comment:
as you receive all claims from the request body, you can use the `anyToken()` method which accepts a map of claims:
```java
mockOAuth2Server.anyToken(
                mockOAuth2Server.issuerUrl(issuerId),
                claims
            ).serialize();
```

Reply:
Yay 😄